pyproject.toml

```diff
@@ -54,6 +55,21 @@ disallow_untyped_defs = true
 no_implicit_optional = false
 exclude = ["python/tests/"]
 
+[tool.pyright]
```

Comment:
Should we pick one of pyright or mypy and drop the other?

Reply:
would agree, would kinda lean towards sticking to mypy rather than making changes. pylsp-mypy exists though I assume it's slower

Reply:
...I've had check-untyped-defs on the entire time I've used mypy. but I have to agree that this is fairly compelling even if the javascript stuff is weird.

Reply:
@yorickvP Thanks for the nudge. I just gave Pyright a look, and found it to be an improvement over mypy. I'd be very happy for us to adopt it across the board. (To that end, I just opened [this PR](https://github.com/replicate/replicate-python/pull/206) in our Python client)